integration-cli/docker_cli_run_test.go

```diff
@@ -1488,7 +1491,7 @@ func TestRunDnsOptions(t *testing.T) {
 }
 
 func TestRunDnsOptionsBasedOnHostResolvConf(t *testing.T) {
```

Comment:
err, meant `TestRunResolvconfUpdater`, but really anything touching the host's `/etc/resolv.conf` is kind of scary since the slightest misstep there can lead to a host that can no longer perform DNS lookups :worried: